designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/MigrationApiEndpoints.scala

```diff
@@ -52,22 +53,42 @@ class MigrationApiEndpoints(auth: EndpointInput[AuthCredentials]) extends BaseEn
       .post
       .in("migrate")
       .in(
-        jsonBody[MigrateScenarioRequestDto].example(
-          Example.of(
-            summary = Some("Migrate given scenario to current Nu instance"),
-            value = MigrateScenarioRequestDtoV1(
-              version = 1,
-              sourceEnvironmentId = "testEnv",
-              remoteUserName = "testUser",
-              processingMode = ProcessingMode.UnboundedStream,
-              engineSetupName = EngineSetupName("Flink"),
-              processCategory = "Category1",
-              scenarioGraph = exampleGraph,
-              processName = ProcessName("test"),
-              isFragment = false
-            )
-          )
-        )
+        jsonBody[MigrateScenarioRequestDto]
+        // FIXME uncomment examples when discriminator validation will work in the NuDesignerApiAvailableToExposeYamlSpec -
```

Comment:
This problem occurred because it's the first DTO with a discriminator within endpoints generated from Tapir definitions. Here's the error in NuDesignerApiAvailableToExposeYamlSpec:
```
postApiMigrate > request > Example0
    $: must be valid to one and only one schema, but 2 are valid with indexes '0, 1'
postApiMigrate > request > Example1
    $: must be valid to one and only one schema, but 2 are valid with indexes '0, 1'
```
I spent some time trying to fix it, but it seems to be more complex than I thought. I wouldn't like to fix it within this PR, so I hid the problematic example. IMO it's a fine workaround for now.